netlify.toml

```diff
@@ -564,3 +564,8 @@
 [[redirects]]
     from = "/docs/tutorials/1-minute/survey"
     to = "/docs/tutorials/survey"
+    
+# Added: 2021-07-14
```

Comment:
Whereas other redirects are because files have likely moved this is more of a functional/feature redirect. So, I'd recommend the comment be extended to identify why this was added so it's not accidentally removed in the future.

Reply:
Added.